packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -83,28 +85,38 @@ class SelectedFilters extends Component {
 			.filter(Boolean);
 	};
 
+	// determines whether any filter has been applied regardless of `showFilter=false`
+	hasFilters = () =>
```

Comment:
If this is to only determine if we have filters applied than I think we should avoid looping to all keys instead we can break the loop if a filter is found. 

Maybe you can use `Array.some`. 

Thoughts @bietkul @lakhansamani